ingest/tests/common/mod.rs

```diff
@@ -0,0 +1,131 @@
+use anyhow::bail;
+use backon::{ExponentialBuilder, Retryable};
+use file_store::file_sink::FileSinkClient;
+use helium_crypto::{KeyTag, Keypair, Network, Sign};
+use helium_proto::services::poc_mobile::{
+    Client as PocMobileClient, VerifiedSubscriberMappingEventReqV1,
+    VerifiedSubscriberMappingEventResV1,
+};
+use ingest::server_mobile::GrpcServer;
+use prost::Message;
+use rand::rngs::OsRng;
+use std::{net::SocketAddr, sync::Arc};
+use tokio::{net::TcpListener, sync::mpsc::Receiver};
+use tonic::{
+    metadata::{Ascii, MetadataValue},
+    transport::Channel,
+    Request,
+};
+use triggered::Trigger;
+
+pub async fn setup_mobile() -> anyhow::Result<(
```

Comment:
any hard reason not to follow the existing test harness setup for the iot ingest ?  It would be preferable to have a common approach across both iot and mobile

Reply:
It seemed cleaner, especially that the mobile stuff needs a lot of file sinks